packages/strapi-admin/admin/src/components/Roles/Tabs/TabsWrapper.js

```diff
@@ -4,6 +4,14 @@ const TabsWrapper = styled.div`
   display: block;
   border-radius: ${({ theme }) => theme.main.sizes.borderRadius};
   box-shadow: ${({ theme }) => `0px 2px 4px 0px ${theme.main.colors.darkGrey}`};
+  .tabs {
```

Comment:
I think you don't need to do it that way. It couples de Flex component to the tab. Instead, what you could do is to put the style in the `Tabs` component.

Something like:

```js
const TabsWrapper = styled.div`
  display: block;
  border-radius: ${({ theme }) => theme.main.sizes.borderRadius};
  box-shadow: ${({ theme }) => `0px 2px 4px 0px ${theme.main.colors.darkGrey}`};

  /* Here */
  ${Tab}:first-of-type {
     // do stuff
  }
`;
```